### DISCUSSION TITLE
Announcing Publii v0.46.2 Release!

### DISCUSSION THREAD

**Discussion by bobmitro** [GetPublii/Publii#1783] [General]
Hey!

We’ve just released Publii CMS v0.46.2! Originally planned as a quick hotfix, this update has grown to include new features and enhancements, such as custom blog post listings, content conversion tools, and expanded UI options. We’ve also added Elixir language support, refined page hierarchy management, and addressed various bug fixes.

For a deep dive into the new features, bug fixes, and improvements, check out the full-release article  [here](https://getpublii.com/blog/release-046-2.html)

As always, we look forward to your feedback and hope these updates improve your experience with Publii!

**Comment by gpsblues** [comment]
Thank you for updating the missing language strings. I had a chance to test it with an updated version of Publii and an older Italian translation: everything now looks very clean and, most importantly, understandable even for less experienced users.

Kudos on the other improvements as well!

**Comment by shawngiese** [comment]
This project is really cool! Looking forward to working more with it!
